tests/vmi_hook_sidecar_test.go

```diff
@@ -189,16 +177,12 @@ var _ = Describe("[sig-compute]HookSidecars", decorators.SigCompute, func() {
 			It("should not start with hook sidecar annotation when the version is not provided", func() {
 				By("Starting a VMI")
 				vmi.ObjectMeta.Annotations = RenderInvalidSMBiosSidecar()
-				vmi, err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(nil)).Create(context.Background(), vmi, metav1.CreateOptions{})
+				vmi = libvmops.RunVMIAndExpectScheduling(vmi, 90)
 				Expect(err).NotTo(HaveOccurred(), "the request to create the VMI should be accepted")
 
 				Eventually(func() bool {
-					vmiPod, exists, err := getVMIPod(vmi)
-					if err != nil {
-						Expect(err).NotTo(HaveOccurred(), "must be able to retrieve the VMI virt-launcher pod")
-					} else if !exists {
-						return false
-					}
+					vmiPod, err := libpod.GetPodByVirtualMachineInstance(vmi, testsuite.GetTestNamespace(vmi))
```

Comment:
Just making sure, when the VM is scheduling does it mean the vmi pod has to exist already?

Reply:
@ShellyKa13 Hi, yep, from code snippet:
```
	// A target Pod exists but is not yet scheduled and in running state.
	Scheduling VirtualMachineInstancePhase = "Scheduling"
```